packages/ckeditor5-list/tests/listcommand.js

```diff
@@ -133,6 +133,33 @@ describe( 'ListCommand', () => {
 				} );
 			} );
 
+			describe( 'options.forceValue', () => {
```

Comment:
I'd make tests showing that lisItem with forceValue true does not change anything, and similarly for paragraphs and false. AFAICS, these tests describe "normal" behavior.

Reply:
There are two assertions for each test to check both scenarios. But I can simplify these tests and split the 4 assertions into 4 tests.

Reply:
:facepalm: